documentation/src/main/asciidoc/topics/con_write_through.adoc

```diff
@@ -1,10 +1,14 @@
 [id='write_through']
 = Write-Through
-Write-Through is an cache writing mode where writes to memory and writes to cache stores are synchronous. When a client application updates a cache entry, in most cases by invoking `Cache.put()`, {brandname} does not return the call until it updates the cache store. This cache writing mode results in updates to the cache store concluding within the boundaries of the client thread.
+Write-Through is an cache writing mode where writes to memory and writes to
+cache stores are synchronous. When a client application updates a cache entry,
+in most cases by invoking `Cache.put()`, {brandname} does not return the call
+until it updates the cache store. This cache writing mode results in updates to
+the cache store concluding within the boundaries of the client thread.
 
 The primary advantage of Write-Through mode is that the cache and cache store
-are updated simultaneously, which ensures that the cache store is always
-consistent with the cache.
+are updated simultaneously, which ensures that the cache store contains all
+entries that are in-memory.
```

Comment:
The original wording is more accurate. If passivation is enabled, then not all entries will be stored in-memory.

Reply:
@ryanemerson @oraNod 
I meant to change or explain the word "cache" as for me "cache" means the complete clustered cache. But in case of a local cache-store (like file) it contains only the entries where this node is the owner.
But maybe this is too technical thinking and might confuse other users